src/components/common/Input/Input.tsx

```diff
@@ -0,0 +1,8 @@
+import { InputProps } from './Input.types';
+import classNames from 'classnames';
+
+const Input: React.FC<InputProps> = (props) => {
+  return <input className={classNames('input input-bordered')} {...props} />;
```

Comment:
classNamesは、特にくっつけたいcssがない限りは使わなくて大丈夫
```suggestion
  return <input className='input input-bordered' {...props} />;
```

例えば、事前にclassを作っておいて、それをくっつけたい時とかに使う
```
const colorClass = "hogehoge"
const textClass = "hugahuga"

return <hoeghoge className={classNames(colorClass, textClass)} />
```